odoo/addons/test_lint/tests/test_markers.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+import os
+import odoo
+from odoo.tests.common import TransactionCase
+from odoo.modules.module import ad_paths
+
+_logger = logging.getLogger(__name__)
+MARKERS = [b'<' * 7, b'>' * 7]
+EXTENSIONS = ('.py', '.js', '.xml', '.less', '.sass')
+
+
+class TestConflictMarkers(TransactionCase):
+
+    def check_file(self, fullpath_name):
+
+        with open(fullpath_name, 'rb') as f:
+            content = f.read()
+            self.assertFalse(any([m in content for m in MARKERS]), 'Conflict markers found in %s' % fullpath_name)
+
+    def test_conflict_markers(self):
+        """ Test that there are no conflict markers left in Odoo files """
+
+        counter = 0
+
+        odoo_path = os.path.dirname(odoo.__file__)
+        paths = ad_paths + [odoo_path]
+        paths.remove(os.path.join(odoo_path, 'addons'))  # avoid checking odoo/addons twice
+
+        for p in paths:
+            for dp, _, file_names in os.walk(p):
+                for fn in file_names:
+                    if any([fn.endswith(ext) for ext in EXTENSIONS]):
```

Comment:
You can pass EXTENSIONS directly to endswith(), no need to iterate. That's why I suggested that.